eng/InstallRuntimes.proj

```diff
@@ -49,21 +58,34 @@
       <PropertyGroup>
         <PowershellWrapper>powershell -NonInteractive -ExecutionPolicy ByPass -NoProfile -command</PowershellWrapper>
         <DotnetInstallScriptCmd>'$(RepositoryEngineeringDir)common\dotnet-install.ps1'</DotnetInstallScriptCmd>
+        <InternalInstallArgs Condition="'$(InternalRuntimeSourceType)' == 'msrc'">-RuntimeSourceFeed $(InternalRuntimeSourceFeed) -RuntimeSourceFeedKey $(InternalRuntimeSourceFeedKey)</InternalInstallArgs>
```

Comment:
Honestly given my comment in the YAML, maybe we can save having an MSRC/Public value (it always tries getting the runtime version from the usual feed, and if it not found there it falls back to whatever feed you give it). So maybe have this be something like:
```xml
<InternalInstallArgs Condition="'$(InternalRuntimeSourceFeed)' != ''"> $(InternalInstallArgs) -RuntimeSourceFeed $(InternalRuntimeSourceFeed)</InternalInstallArgs>
<InternalInstallArgs Condition="'$(InternalRuntimeSourceFeedKey)' != ''"> $(InternalInstallArgs) -RuntimeSourceFeedKey $(InternalRuntimeSourceFeedKey)</InternalInstallArgs>
```
That way we support other feeds (both authenticated and non-authenticated). 

If we do this remove `InternalRuntimeSourceType` in the YAML, build scripts, and then change line 34 above to:
```xml
<InternalReleaseTesting Condition="'$(InternalRuntimeSourceFeed)' != ''">true</InternalReleaseTesting> 
```

Reply:
Also, these arguments are not case sensitive so they can be shared for windows and linux/osx in line 34

Reply:
But these properties (both the *Feed and *Key) are always set by the yaml (to the msrc variable group) because the conditional on *Type didn't work. Only Type == "msrc" means use these variables. Type == "public" means just use *Version to run the tests only on that versions and Type == anything else means run normally.

I guess now I see why renaming the command line options and properties might make more sense give what I just said about Type == "msrc".